inc/relation.constant.php

```diff
@@ -1639,6 +1641,9 @@
     'glpi_wifinetworks' => [
         'glpi_networkportwifis' => 'wifinetworks_id',
     ],
+    'glpi_webhooks' => [
+        '_glpi_queuedwebhooks' => 'webhooks_id',
```

Comment:
You should add the following method to `Webhook`, to automatically delete all queued webhooks when a webhook is deleted from DB.

```php
    public function cleanDBonPurge()
    {

        $this->deleteChildrenAndRelationsFromDb(
            [
                QueudWebhook::class,
            ]
        );
    }
```